maintenance/Maintenance.rdf

```diff
@@ -242,303 +268,445 @@
         <owl:equivalentClass>
             <owl:Class>
                 <owl:intersectionOf rdf:parseType="Collection">
-                    <rdf:Description rdf:about="https://dev.industrialontologies.org/ontology/maintenance/MaintenanceReferenceOntology/UndesireableDisposition" />
+                    <rdf:Description rdf:about="https://dev.industrialontologies.org/ontology/maintenance/MaintenanceReferenceOntology/UndesireableDisposition"/>
                     <owl:Restriction>
-                        <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/BFO_0000054" />
-                        <owl:someValuesFrom rdf:resource="https://dev.industrialontologies.org/ontology/maintenance/MaintenanceReferenceOntology/FailureProcess" />
+                        <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/BFO_0000054"/>
+                        <owl:someValuesFrom rdf:resource="https://dev.industrialontologies.org/ontology/maintenance/MaintenanceReferenceOntology/FailureProcess"/>
                     </owl:Restriction>
                 </owl:intersectionOf>
             </owl:Class>
         </owl:equivalentClass>
-        <rdfs:subClassOf rdf:resource="https://dev.industrialontologies.org/ontology/maintenance/MaintenanceReferenceOntology/UndesireableDisposition" />
+        <rdfs:subClassOf rdf:resource="https://dev.industrialontologies.org/ontology/maintenance/MaintenanceReferenceOntology/UndesireableDisposition"/>
         <rdfs:label>disposition to fail</rdfs:label>
-        <iof-av:firstOrderLogicDefinition>UndesireableDisposition(x) ∧ ∃y(FailureProcess(y) ∧ hasRealization(x,y)) ↔ DispositionToFail(x)</iof-av:firstOrderLogicDefinition>
-        <iof-av:naturalLanguageDefinition>undesirable disposition of an item causing it to fail as a result of a failure process</iof-av:naturalLanguageDefinition>
-        <iof-av:semiFormalNaturalLanguageDefinition>&apos;disposition to fail&apos;: every instance of &apos;disposition to fail&apos; is defined as exactly an instance of &apos;undesirable disposition&apos; that &apos;has realization&apos; some &apos;failure process&apos;</iof-av:semiFormalNaturalLanguageDefinition>
+        <skos:example>disposition to explode, disposition to fracture, disposition to sieze</skos:example>
+        <iof-av:firstOrderLogicDefinition>DispositionToFail(x) ∨ DispositionToExhibitUndesirableBehaviour ↔ UndesirableDisposition(x)</iof-av:firstOrderLogicDefinition>
```

Comment:
This appears to be the axiom for undesirable disposition and not disposition to fail